src/components/AddFreakModal/AddFreakModal.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Button } from 'react-bootstrap';
+import Modal from '../Modal/Modal';
+import EditFreakForm from '../EditFreakForm/EditFreakForm';
+import { FreakModelDefault } from '../../models/freaks';
+
+function AddFreakModal({ title, isOpen, onClose }) {
+  const [ freak, setFreak ] = useState(FreakModelDefault);
+
+  return (
+    <Modal
+      title={ title }
+      isOpen={ isOpen }
+      headerContent={ <button className="filter__button-reset" type="button">Reset</button> }
+      footerContent={ <Button className="filter__button-apply" variant="primary py-2 px-5 ">Submit</Button> }
```

Comment:
The variant is not made for you to put any class name you want, keep the classes for padding in the className field, variant is for `variants`.